nav-kibana-dev.docnav.json

```diff
@@ -177,6 +177,9 @@
         },
         {
           "id": "kibDevTutorialScreenshotting"
+        },
+        {
+          "id": "kibDevTutorialsServerlessProjectNavigation"
```

Comment:
I am adding to the "tutorials" sections because couldn't find where to fit it better. 
Maybe we should add "Serverless" section?